File: calend.py
#!/usr/bin/env python3
# sudo pip3 install feedparser

import locale
import time

import feedparser


def get_today():
    f = feedparser.parse('http://www.calend.ru/img/export/today-holidays.rss')
    locale.setlocale(locale.LC_TIME, "ru_RU")
    now = "%s %s" % (int(time.strftime("%d")), time.strftime("%B"))

    hol = now + ": "
    for i in f.entries:
        if now in i['title']:
            hol = hol + i['title'].split('-')[1].strip() + "; "
    return hol[:-2]  # remove trailing ;


def main():
    print(get_today())


if __name__ == "__main__":
    main()

File: calend.go
package main

// go get github.com/mmcdole/gofeed

import (
	"fmt"
	"strconv"
	"strings"
	"time"

	"github.com/mmcdole/gofeed"
)

func main() {
	fp := gofeed.NewParser()
	time.LoadLocation("Europe/Moscow")
	feed, _ := fp.ParseURL("http://www.calend.ru/img/export/today-holidays.rss")
	//fmt.Println(er)

	today := strconv.Itoa(time.Now().Day())
	text := ""
	// fmt.Println(feed)
	for _, element := range feed.Items {
		if strings.HasPrefix(element.Title, today) {
			if len(text) == 0 {
				t := strings.Split(element.Title, " ")
				text = t[0] + " " + t[1] + ": "
			}
			text = text + element.Title[strings.Index(element.Title, "- ")+2:] + "; "
		}
	}
	fmt.Println(text[:len(text)-2])
}
